### ISSUE TITLE
Lists entry in main popup menu doesn't work on Android

### ISSUE THREAD

**Issue by timbray** [cheeaun/phanpy#473]
Hi, I bring up the main menu on my Pixel 7 and it looks as it should, but when I tap on Lists it does nothing.  On desktop it works fine, sliding out a selector with "All lists" or the one list I have.

![Screenshot_20240327-180010](https://github.com/cheeaun/phanpy/assets/17555/37cb1f3f-fb54-4d9c-9239-5563cd6f741d)

**Comment by cheeaun** [comment]
@timbray does the 'More…' menu work for you?

**Comment by timbray** [comment]
> @timbray does the 'More…' menu work for you?

Nope, that's also a no-op now. BTW also on my big Samsung tablet it's the same.

**Comment by cheeaun** [comment]
@timbray I've made some potential fixes (I couldn't reproduce this on my side - Pixel 3, Android 12), could you check if it's still happening?

**Comment by timbray** [comment]
Seems a bit better on the Samsung tablet, I can get the lists and more menus to slide out but I have to sort of pull-right on them.

**Comment by cheeaun** [comment]
@timbray sorry what do you mean by "pull-right on them"?

**Comment by timbray** [comment]
Oops, turns out that what I meant was “long press”.  Long press on them and they work, but not on tap.  Both on the Sammy tablet and the Pixel 7.

**Comment by cheeaun** [comment]
@timbray is this on dev or prod?

**Comment by timbray** [comment]
Prod.

**Comment by marcin-kozinski** [comment]
I have the same issue. Long-press doesn't seem to work for me.

Android 14; Pixel 7a Build/AP1A.240305.019.A1
Chrome 123.0.6312.80

**Comment by cheeaun** [comment]
@marcin-kozinski does tapping (not long-press) work?

**Comment by marcin-kozinski** [comment]
No, it doesn't. Sorry I wasn't clear!

**Comment by marcin-kozinski** [comment]
Sorry, just checked dev again and I saw the "new update available" message in the main menu. After reloading sub menus now work!

**Comment by timbray** [comment]
Confirmed, fixed in dev.
